frontend/src/components/housing/enquiry/EnquiryListNavigator.vue

```diff
@@ -31,11 +41,27 @@ const { loading, enquiries } = defineProps<{
 const emit = defineEmits(['enquiry:delete']);
 
 // State
+const route = useRoute();
 const selection: Ref<Enquiry | undefined> = ref(undefined);
+const pagination: Ref<Pagination> = ref({});
+
+// read from query params if tab is set to enquiry otherwise use default values
+if (route.query.tab === '1') {
+  pagination.value.rows = toNumber(route.query.rows as string) ?? 10;
+  pagination.value.order = toNumber(route.query.order as string) ?? -1;
+  pagination.value.field = (route.query.field as string) ?? 'submittedAt';
+  pagination.value.page = toNumber(route.query.page as string) ?? 0;
+} else {
+  pagination.value.rows = 10;
+  pagination.value.order = -1;
+  pagination.value.field = 'submittedAt';
+  pagination.value.page = 0;
+}
```

Comment:
Style: This could potentially be simplified by doing the following instead - should be logically equivalent:
```ts
const pagination: Ref<Pagination> = ref({
  rows: 10,
  order: -1,
  field: 'submittedAt',
  page: 0
});

// read from query params if tab is set to enquiry otherwise use default values
if (route.query.tab === '1') {
  pagination.value.rows = toNumber(route.query.rows as string) ?? 10;
  pagination.value.order = toNumber(route.query.order as string) ?? -1;
  pagination.value.field = (route.query.field as string) ?? 'submittedAt';
  pagination.value.page = toNumber(route.query.page as string) ?? 0;
}
```